pkg/sidecar-controller/groupsnapshot_helper.go

```diff
@@ -423,6 +423,11 @@ func (ctrl *csiSnapshotSideCarController) createGroupSnapshotWrapper(groupSnapsh
 		creationTime = time.Now()
 	}
 
+	groupSnapshotSecret, err := utils.GetSecretReference(utils.SnapshotterSecretParams, class.Parameters, groupSnapshotContent.GetObjectMeta().GetName(), nil)
```

Comment:
instead of `utils.SnapshotterSecretParams` can use have group specific secrets? like below

```go
PrefixedGroupSnapshotterSecretNameKey   = csiParameterPrefix + "group-snapshotter-secret-name"      // Prefixed name key for CreateGroupSnapshot secret
PrefixedGroupSnapshotterSecretNamespaceKey = csiParameterPrefix + "group-snapshotter-secret-namespace" // Prefixed namespace key for DeleteGroupSnapshot secret
var GroupSnapshotterSecretParams = secretParamsMap{
	name:               "Snapshotter",
	secretNameKey:      PrefixedGroupSnapshotterSecretNameKey,
	secretNamespaceKey: PrefixedGroupSnapshotterSecretNamespaceKey,
}
```

Reply:
I am not sure that there is much  value in having separate GroupSnapshot secrets. @RaunakShah or @xing-yang what do you think?